shared-libs/environment/test/index.spec.js

```diff
@@ -74,4 +80,98 @@ describe('environment', () => {
       }
     });
   });
+
+  describe('deploy info', () => {
+    beforeEach(() => {
+      stubProcessEnv({ COUCH_URL: 'http://admin:pass@localhost:5984/medicdb' });
+    });
+
+    afterEach(() => {
+      sinon.restore();
+      process.env = originalEnv;
+    });
+
+    describe('getDeployInfo', () => {
+      it('should return deploy info with version from build_info', async () => {
+        const ddoc = {
+          _id: '_design/medic',
+          build_info: {
+            version: '4.1.0',
+            date: '2023-01-01',
+            base_version: '4.0.0'
+          },
+          deploy_info: {
+            timestamp: '2023-01-02'
+          }
+        };
+
+        sinon.stub(request, 'get').resolves(ddoc);
+        const env = rewire('../src/index');
+
+        const result = await env.getDeployInfo();
+        expect(result).to.deep.equal({
+          version: '4.1.0',
+          date: '2023-01-01',
+          base_version: '4.0.0',
+          timestamp: '2023-01-02'
+        });
+        expect(request.get.callCount).to.equal(1);
+        expect(request.get.args[0][0].url).to.equal('http://admin:pass@localhost:5984/medicdb/_design/medic');
```

Comment:
I think we should be more strict about the assertion here, and assert all params that are passed to request.get. 
We could easily make a `DELETE` request and the test would still pass